### PR TITLE
Enhancing hasMeta query scope with array support

### PR DESCRIPTION
**Pull request by lasmanis**
One can now pass an array of values to the hasMeta query scope, and get all posts that match.

For instance, `Corcel\Post::hasMeta('city', ['amsterdam', 'london'])->get();` will return all posts that have 'amsterdam' or 'london' as `meta_value` for the 'city' `meta_key`.

----

Unit test results:
PHPUnit 5.7.20 by Sebastian Bergmann and contributors.

.............................................................     61 / 61 (100%)

Time: 1.02 seconds, Memory: 14.75MB

OK (61 tests, 163 assertions)

### CODE REVIEW

File: tests/HasMetaTest.php [hunk 1]
```diff
@@ -9,6 +9,17 @@ public function testUserHasMeta()
         $post = Post::published()->hasMeta('username', 'juniorgrossi')
             ->first();
 
+        $posts = Post::published()->hasMeta('city', ['amsterdam', 'london'])
```
Comment (jgrossi) [inline tests/HasMetaTest.php]: Hey @lasmanis did you changed the testing database to include the `city` taxonomy and `amsterdam` and `londom` terms?

Comment (jgrossi) [inline tests/HasMetaTest.php]: Update the `database.sql` file in `tests/config` with your local sql dump, to make this test passing.

Comment (lasmanis) [inline tests/HasMetaTest.php]: I just did. These are the results now:

```
PHPUnit 5.7.20 by Sebastian Bergmann and contributors.

.............................................................     61 / 61 (100%)

Time: 827 ms, Memory: 14.75MB

OK (61 tests, 164 assertions)
```

**Comment by jgrossi** [comment]
Thanks @lasmanis!

**Pull request merged**